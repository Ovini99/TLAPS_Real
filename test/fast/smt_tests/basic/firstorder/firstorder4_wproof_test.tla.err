@!!BEGIN
@!!type:warning
@!!msg:File "./firstorder4_wproof_test.tla", line 1, character 1 to line 23, character 4 :
Warning: module name "firstorder4_test" does not match file name "firstorder4_wproof_test.tla".
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:21:5:21:7
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:21:13:21:17
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:3
@!!loc:21:19:21:23
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:4
@!!loc:15:5:15:12
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:5
@!!loc:17:5:17:7
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:6
@!!loc:17:8:17:12
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:7
@!!loc:19:5:19:7
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:8
@!!loc:19:8:19:12
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:8
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:21:13:21:17
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:3
@!!loc:21:19:21:23
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

** Unexpanded symbols: ---

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:6
@!!loc:17:8:17:12
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:8
@!!loc:19:8:19:12
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:21:5:21:7
@!!status:proved
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!already:false
@!!END

@!!BEGIN
@!!type:obligation
@!!id:4
@!!loc:15:5:15:12
@!!status:proved
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!already:false
@!!END

@!!BEGIN
@!!type:obligation
@!!id:5
@!!loc:17:5:17:7
@!!status:proved
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!already:false
@!!END

@!!BEGIN
@!!type:obligation
@!!id:7
@!!loc:19:5:19:7
@!!status:proved
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!already:false
@!!END

File "./firstorder4_wproof_test.tla", line 1, character 1 to line 23, character 4:
[INFO]: All 8 obligations proved.
