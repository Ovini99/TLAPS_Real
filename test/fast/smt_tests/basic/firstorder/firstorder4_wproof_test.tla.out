
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:17:42 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 --nofp firstorder4_wproof_test.tla --stretch 5

(* created new ".tlacache/TLAPS.tlaps/TLAPS.thy" *)
(* fingerprints written in ".tlacache/TLAPS.tlaps/fingerprints" *)
(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from the file `.tlacache/firstorder4_wproof_test.tlaps/fingerprints`, in order to overwrite with the new fingerprints, and then save the results at the end. *)
COMMAND-LINE ARGUMENT
file=.tlacache/firstorder4_wproof_test.tlaps/tlapm_ffa708.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/firstorder4_wproof_test.tlaps/tlapm_ffa708.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #1
;;   generated from file "./firstorder4_wproof_test.tla", line 21, characters 5-6

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun boolify (u) Bool)

;; Terms, predicates and strings
(declare-fun a_CONSTANTunde_Dunde_a (u) u)
(declare-fun a_CONSTANTunde_yunde_a () u)


;; Theorem to be Proved
(assert (not false))
(assert false)
(assert (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a)))

(check-sat)
(exit)

COMMAND-LINE ARGUMENT
file=.tlacache/firstorder4_wproof_test.tlaps/tlapm_3c6941.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/firstorder4_wproof_test.tlaps/tlapm_3c6941.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #4
;;   generated from file "./firstorder4_wproof_test.tla", line 15, characters 5-11

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun boolify (u) Bool)

;; Terms, predicates and strings
(declare-fun a_CONSTANTunde_Dunde_a (u) u)


;; Theorem to be Proved
(assert (not
  (exists ((a_CONSTANTunde_xunde_a u))
    (=>
      (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_xunde_a))
      (forall ((a_CONSTANTunde_yunde_a u))
        (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a)))))))
(assert
  (=>
    (forall ((a_CONSTANTunde_xunde_a u))
      (and
        (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_xunde_a))
        (exists ((a_CONSTANTunde_yunde_a u))
          (not (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a))))))
    false))

(check-sat)
(exit)

COMMAND-LINE ARGUMENT
file=.tlacache/firstorder4_wproof_test.tlaps/tlapm_7fb769.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/firstorder4_wproof_test.tlaps/tlapm_7fb769.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #5
;;   generated from file "./firstorder4_wproof_test.tla", line 17, characters 5-6

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun boolify (u) Bool)

;; Terms, predicates and strings
(declare-fun a_CONSTANTunde_Dunde_a (u) u)


;; Theorem to be Proved
(assert (not
  (exists ((a_CONSTANTunde_yunde_a u))
    (not (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a))))))
(assert
  (forall ((a_CONSTANTunde_xunde_a u))
    (and
      (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_xunde_a))
      (exists ((a_CONSTANTunde_yunde_a u))
        (not (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a)))))))

(check-sat)
(exit)

COMMAND-LINE ARGUMENT
file=.tlacache/firstorder4_wproof_test.tlaps/tlapm_dd605e.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/firstorder4_wproof_test.tlaps/tlapm_dd605e.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #7
;;   generated from file "./firstorder4_wproof_test.tla", line 19, characters 5-6

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun boolify (u) Bool)

;; Terms, predicates and strings
(declare-fun a_CONSTANTunde_Dunde_a (u) u)
(declare-fun a_CONSTANTunde_yunde_a () u)


;; Theorem to be Proved
(assert (not (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_a))))
(assert
  (forall ((a_CONSTANTunde_xunde_a u))
    (and
      (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_xunde_a))
      (exists ((a_CONSTANTunde_yunde_unde_2a u))
        (not (boolify (a_CONSTANTunde_Dunde_a a_CONSTANTunde_yunde_unde_2a)))))))

(check-sat)
(exit)

RESULT
unsat

RESULT
unsat

RESULT
unsat

RESULT
unsat

(* created new ".tlacache/firstorder4_wproof_test.tlaps/firstorder4_wproof_test.thy" *)
(* fingerprints written in ".tlacache/firstorder4_wproof_test.tlaps/fingerprints" *)
